### PR TITLE
Swagger spec improvements

### PR DESCRIPTION
**Pull request by goddammit**
This PR addresses #

Proposed changes include:
* describe the functionality changes
* describe new or updated tests
* describe changes to the documentation

### CODE REVIEW

**Comment by Nyco** [comment]
All good to me, just please think of [skip ci] when touching only docs (no code), and fill your PR description.
Thanks!

File: doc/http-api/backend_swagger.yml [hunk 1]
```diff
@@ -3,13 +3,14 @@ info:
...
+    Explore MongooseIM features using our REST API.
+
+    If you are familiar with the `mongooseimctl` command, the provided control functionality will seem very similar.
     The response objects are modeled on the `ejabberd_commands` entries and print-outs from `mongooseimctl`.
 
-    In many cases fields like **username** or **caller** expect **JID** (jabber identity, f.e. **alice@wonderland.com**).
-    There are two types of **JIDs**:
-      * **bare JID** - consists of **username** and **domain name** (XMPP host, usually the one which is set in your `ejabberd.cfg` file).
-      * **full JID** - this is **bareJID** with online user's resource to uniquely identify user's connection (f.e. **alice@wonderland.com/resource**).
+    Please note that many of the fields such as **username** or **caller** expect a **JID** (jabber identity, f.e. **alice@wonderland.com**). There are two types of **JIDs**:
```
Comment (Nyco) [inline doc/http-api/backend_swagger.yml]: Please change "jabber identity" for "Jabber identifier"

File: doc/http-api/backend_swagger.yml [hunk 2]
```diff
@@ -3,13 +3,14 @@ info:
...
+      * **bare JID** - consists of **username** and **domain name** (XMPP host, usually the one set in your `ejabberd.cfg` file).
+      * **full JID** - is a **bareJID** with online user's resource to uniquely identify user's connection (f.e. **alice@wonderland.com/resource**).
```
Comment (Nyco) [inline doc/http-api/backend_swagger.yml]: Not "bareJID", but rather "bare JID".

File: doc/http-api/client_swagger.yml [hunk 3]
```diff
@@ -213,7 +213,7 @@ securityDefinitions:
   mongoose_basic_auth:
     type: basic
     description: |
-      Log in using your user's bare JID (F.e. alice@wonderland.com) and their password.
+      Log in using your userid (users's bare JID - F.e. alice@wonderland.com) and their password.
```
Comment (Nyco) [inline doc/http-api/client_swagger.yml]: Not if "F.e." or rather "f.e.".

File: doc/http-api/backend_swagger.yml (hunk 1 above)
Comment (michalwski) [inline doc/http-api/backend_swagger.yml]: Agree, @goddammit could you change this?

File: doc/http-api/backend_swagger.yml (hunk 2 above)
Comment (michalwski) [inline doc/http-api/backend_swagger.yml]: Agree, @goddammit?

**Pull request merged**